vault/dynamic_system_view.go

```diff
@@ -6,6 +6,7 @@ import (
 	"time"
 
 	"github.com/hashicorp/errwrap"
+	"github.com/hashicorp/vault/sdk/helper/random"
 
```

Comment:
nit: shouldn't line 9 be down in line 16?

Reply:
Fixed